distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1926,6 +1926,14 @@ private void persistToWorkspace(int size) throws Exception {
 
     @Test
     public void testContentVersioning() throws Exception {
+
+        Configuration config = getAdminConfig().getConfiguration("ddf.catalog.history.Historian");
+        config.setBundleLocation(
+                "mvn:ddf.catalog.core/catalog-core-standardframework/2.10.0-SNAPSHOT");
```

Comment:
This is going to be very fragile since the version will change with the next release.  You should be able to lookup the config via the service PID.